about.html

```diff
@@ -0,0 +1,123 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <link rel="stylesheet" href="CSS/style.css">
+  <link rel="stylesheet" href="CSS/About.css">
+  <link rel="stylesheet" href="CSS/media.css">
+  <title>MMF 2022 | Mawazine Music Festival</title>
+  <link rel="icon" type="image/png" href ="/Assets/mawazine.png">
+</head>
+<body>
+  <header>
+    <div class="menu">
+      <div></div>
+    </div>
+    <nav class="first">
+      <ul>
+        <li><a href=""><i class="fa fa-twitter fa-x"></i></a></li>
+        <li><a href=""><i class="fa fa-facebook fa-x"></i></a></li>
+        <li> <select name="languages" id="lang">
+          <option value="English" selected>English</option>
+          <option value="Arabic">Arabic</option>
+          <option value="Spanish">Spanish</option>
+          <option value="French">French</option>
+        </select></li>
+        <li>My page</li>
+        <li>logout</li>
+      </ul>
+    </nav>
+    <div class="second">
+      <img class="logo" src="Assets/mawazine.png" alt="logo">
+      <nav class="nav-bar">
+        <ul>
+          <li><a href="index.html" class="nav-link" id="Home">Home</a></li>
+          <li><a href="#" class="nav-link">About</a></li>
+          <li><a href="#Program" class="nav-link">Program</a></li>
+          <li><a href="#Geusts" class="nav-link" >Guests</a></li>
+          <li><a class="nav-link" href="#Parteners">Partners</a></li>
+          <li><a href="#Envents" class="nav-link">Event</a></li>
+          <li><a href="#" class="nav-link"><span id="mawazine">mawazine</span> </a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main>
+    <div class="head">
+      <p class="slogan">"Largest Music Festival in Africa"</p>
+      <h1 class="title">
+        Mawazine Music <br>
+      <span class="title-content">Festival</span> <br>
+      <span class="title-content">2022</span></h1>
+    </div>
+    <div class="description">
+      <p>The Mawazine Music Festival is the largest conference dedicated to music in the Africa. Held every year in Rabat Morroco, the conference features speakers from the music industry, musicians, and music education professionals. This year’s conference will be held on July 25-27 at the same place.</p>
+    </div>
+    <div class="contact">
+      <p>Please contact us per Email for any further questions about Mawazine Music Festifal 2022!</p>
+      <a href="#">mawazinerabat@gmail.com</a>
+    </div>
+  </main>
+  <section class="log-mmf">
```

Comment:
Consider matching the logo section as per the Original design. Align the text in the center as in the original design.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/182564990-c65311fe-062b-44a6-9929-c4825cd0dee3.png) | ![image](https://user-images.githubusercontent.com/33268927/182565200-c53bd8c9-7ab0-425b-994c-89776f287a20.png) |